src/app/users/user-profile/user-profile.component.html

```diff
@@ -0,0 +1,54 @@
+<h3 i18n>Member Profile</h3>
+<form [formGroup]="editForm" (ngSubmit)="onSubmit()">
+  <mat-grid-list cols="3">
+    <mat-form-field>
+      <input matInput i18n-placeholder placeholder="First Name" formControlName="firstName" required>
```

Comment:
You should use `planet-form-error-messages` here like in the `courses-add` component.

Reply:
@Rupesh87  Could you please use error message ?